pages/_app.tsx

```diff
@@ -57,6 +60,7 @@ const App = ({ Component, pageProps }: AppProps) => { // eslint-disable-line
                     src={`https://widget.usersnap.com/global/load/${USERSNAP_GLOBAL_API_KEY}?onload=onUsersnapCXLoad`}
                 />
                 <script defer data-domain="pools.tracer.finance" src="https://plausible.io/js/plausible.js" />
+                <link rel="stylesheet" href="https://use.typekit.net/klm0viv.css" />
```

Comment:
is the 'aileron' font coming from issy and the designers?

Reply:
From Declan, yes